internal/arr/collector/radarr.go

```diff
@@ -72,6 +73,12 @@ func NewRadarrCollector(c *config.ArrConfig) *radarrCollector {
 			[]string{"quality"},
 			prometheus.Labels{"url": c.URL},
 		),
+		movieTags: prometheus.NewDesc(
+			"radarr_movie_tag_total",
+			"Tags by total",
```

Comment:
Same here, can we match the other metrics?

"Total number of downloaded movies by tag"

Reply:
Updated.